realm/realm-library/build.gradle

```diff
@@ -217,22 +218,19 @@ dependencies {
     api "io.realm:realm-annotations:${version}"
     implementation 'com.google.code.findbugs:jsr305:3.0.2'
     implementation 'com.getkeepsafe.relinker:relinker:1.4.0'
-    implementation "org.mongodb:bson:${properties.getProperty('BSON_DEPENDENCY_VERSION')}"
-    implementation('io.reactivex.rxjava2:rxandroid:2.1.1') {
+    implementation "org.mongodb:bson:${properties.getProperty('BSON_DEPENDENCY')}"
```

Comment:
As far as I remember there are some of the types from this library that is actually part of our API (Decoder, CodecRegistry, etc)...at least for the sync flavor, in which case it would be more correct to have this as an `api` dependency.

Reply:
makes sense 👌👌